training/cogvideox_text_to_video_sft.py

```diff
@@ -744,7 +744,7 @@ def load_model_hook(models, input_dir):
             last_lr = lr_scheduler.get_last_lr()[0] if lr_scheduler is not None else args.learning_rate
```

Comment:
Should the OFS related changes be propagated here too?

Reply:
OFS only exists in image to video